src/script_opt/CPP/Func.h

```diff
@@ -85,6 +77,7 @@ class CPPLambdaFunc : public ScriptFunc {
     // Methods related to sending lambdas via Broker.
     std::optional<BrokerData> SerializeCaptures() const override;
     void SetCaptures(Frame* f) override;
+    void SetCapturesVec(std::unique_ptr<std::vector<ZVal>> _captures_vec) { captures_vec = std::move(_captures_vec); }
```

Comment:
I'm not seeing where this is used in the context of this PR and am also thinking this could just go into `class Func`, I'll do that as a fixup and make the field private again. If it breaks something, you might need to add a fixup.